### DISCUSSION TITLE
Pomiary ze smogomierza na www

### DISCUSSION THREAD

**Discussion by hejsiri** [hackerspace-silesia/Smogomierz#137] [General]
Zrobiłem taką prostą stronę do prezentacji pomiarów z czujników smogomierza na www. Co o tym sądzicie? beta 1 :-)

http://dworcowa.paslek.com

<img width="1028" alt="zrzut ekranu 2019-03-5 o 17 56 08" src="https://user-images.githubusercontent.com/37380727/53822435-025c4800-3f70-11e9-8ccb-ddac8ce34674.png">

Link do paczki ze stroną: [www.dworcowa.paslek.com/smogomierz-www.zip](http://www.dworcowa.paslek.com/smogomierz-www.zip)
Wystarczy wpisać swój adres smogomierza w index.php i współrzędne w wykresach.

**Comment by ookris** [comment]
No to ja też się pochwalę. Co prawda nic odkrywczego nie stworzyłem (tak jak kolega powyżej) ale pochwalić się mogę :P

http://bolecair.info/
